ai-ml/trainium-inferentia/eks.tf

```diff
@@ -135,7 +134,6 @@ module "eks" {
 
       tags = merge(local.tags, {
         Name                     = "core-node-grp",
-        "karpenter.sh/discovery" = local.name
```

Comment:
Do you need to remove the comma from the above line? Could you run pre-commit locally on this PR please?

Reply:
Done